modules.css

```diff
@@ -0,0 +1,137 @@
+/*  Header MODULE
+–––––––––––––––––––––––––––––––––––––––––––––––––– */
+.pageHeader {
+  border: 13px solid;
+  border-width: 0px 0px 13px;
+  -moz-border-image: url(imgs/break.png) 0 0 13 repeat;
+  -webkit-border-image: url(imgs/break.png) 0 0 13 repeat;
+  -o-border-image: url(imgs/break.png) 0 0 13 repeat;
+  border-image: url(imgs/break.png) 0 0 13 fill repeat;
+  padding-bottom: 30px; }
```

Comment:
I would keep this here. It looks like you have a repeat of this border styling in "Base"